### PR TITLE
TP53 module modified to take the right path

### PR DESCRIPTION
**Pull request by runjin326**
#### What scientific question is your analysis addressing?
Modify TP53 module to use the right path of status file.

#### What was your approach?

1) Modify the output dir for the 2 status files
2) Modify the input dir for the 2 status files 
3) Modify `README.md` file to reflect the change 
4) Modify the bash script to remove unnecessary lines

#### What GitHub issue does your pull request address?
NA

#### Which areas should receive a particularly close look?
NA

#### Is there anything that you want to discuss further?
No

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes.

#### What types of results are included (e.g., table, figure)?
NA

#### What is your summary of the results?
NA

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by runjin326** [comment]
Note - this is not staggered so I will do a dev merge if we merge focal CN path file first.

File: analyses/tp53_nf1_score/run_classifier.sh [hunk 1]
```diff
@@ -78,9 +78,4 @@ Rscript -e "rmarkdown::render('${analysis_dir}/05-tp53-altered-annotation.Rmd',p
 # evaluate classifer scores for stranded data
 python3 ${analysis_dir}/06-evaluate-classifier.py -s ${analysis_dir}/results/tp53_altered_status.tsv -f ${analysis_dir}/results/gene-expression-rsem-tpm-collapsed_classifier_scores.tsv -c ${histology_file} -r "PBTA"
 
-# Skip poly-A steps in CI
-if [ "$POLYA" -gt "0" ]; then
-  python3 ${analysis_dir}/06-evaluate-classifier.py -s ${analysis_dir}/results/tp53_altered_status.tsv -f ${analysis_dir}/results/gene-expression-rsem-tpm-collapsed_classifier_scores.tsv -c ${histology_file} -r "PBTA"
-fi
-
```
Comment (jharenza) [inline analyses/tp53_nf1_score/run_classifier.sh:85]: I think we need to probably update this module later to assess the ROC separately for polyA and stranded, but just noting that for now.

File: analyses/tp53_nf1_score/00-tp53-nf1-alterations.R [hunk 2]
```diff
@@ -93,10 +93,7 @@ tp53_coding <- coding_consensus_snv %>%
 # subset to TP53 cnv loss and format to tp53_coding file format
 tp53_loss<-cnvConsesus %>%
```
Comment (jharenza) [inline analyses/tp53_nf1_score/00-tp53-nf1-alterations.R]:
```suggestion
tp53_loss<-cnvConsensus %>% 
```

noticing some old typos - can you update while you are here?

**Comment by runjin326** [comment]
@jharenza - now the dev is merged and this is ready for review. I think we can merge this first and then when we have data ready for the release (in this case, the status files from focal cn rerun), we will then do the re-run PR.

File: analyses/tp53_nf1_score/00-tp53-nf1-alterations.R (hunk 2 above)
Comment (jharenza) [reply analyses/tp53_nf1_score/00-tp53-nf1-alterations.R]: @runjin326 can you update the typos in this script please

Comment (runjin326) [reply analyses/tp53_nf1_score/00-tp53-nf1-alterations.R]: Sure!

Comment (runjin326) [reply analyses/tp53_nf1_score/00-tp53-nf1-alterations.R]: @jharenza - ready for another look now!

**Review by jharenza — APPROVED** [review]

**Pull request merged**